### PR TITLE
arcanist 20180215

### PR DESCRIPTION
**Pull request by lexfrei**
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-php/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/arcanist.rb [hunk 1]
```diff
@@ -6,14 +6,14 @@ class Arcanist < Formula
   revision 1
```
Comment (SMillerDev) [inline Formula/arcanist.rb]: this needs to be removed

**Comment by javian** [comment]
Hmm, it looks like an additional number was added to the version of the these formulas so the new version number is lower than the original one. Can you add a 1 at the end of the current version ?

File: Formula/arcanist.rb [hunk 2]
```diff
@@ -3,17 +3,16 @@
...
   homepage "https://secure.phabricator.com/book/phabricator/article/arcanist/"
-  revision 1
 
   stable do
-    url "https://github.com/wikimedia/arcanist/archive/release/2017-11-29/1.tar.gz"
-    sha256 "33eef2c3b88f143f7b87e121e54b9a10c7c976fe647ec5d65276458a54e43a25"
-    version "201711291"
+    url "https://github.com/wikimedia/arcanist/archive/release/2018-02-15/1.tar.gz"
+    sha256 "75888393e3f0ffb8a567a877b31903db8390b622f73ad501b8eb1bcafb4320fa"
+    version "20180215"
```
Comment (javian) [inline Formula/arcanist.rb]: Can you add a 1 at the end of the current version

File: Formula/arcanist.rb [hunk 3]
```diff
@@ -3,17 +3,16 @@
...
 
     resource "libphutil" do
-      url "https://github.com/wikimedia/phabricator-libphutil/archive/release/2017-11-29/1.tar.gz"
-      sha256 "d43338b02e24a91cf93b30da4f7d2a971d3e185b170df2bb490b7e01a4452524"
-      version "201711291"
+      url "https://github.com/wikimedia/phabricator-libphutil/archive/release/2018-02-15/1.tar.gz"
+      sha256 "a46f0721fa8166ed5edf24ecfe395d7d41e4728681969dded12c664c6fb2074e"
+      version "20180215"
```
Comment (javian) [inline Formula/arcanist.rb]: Can you add a 1 at the end of the current version

**Comment by javian** [comment]
Thanks @lexfrei

**Pull request closed without merging**